VideoLocker/src/main/java/org/edx/mobile/user/GetCourseEnrollmentsTask.java

```diff
@@ -0,0 +1,34 @@
+package org.edx.mobile.user;
+
+/**
+ * Created by cleeedx on 4/7/16.
+ */
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import com.google.inject.Inject;
+
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.task.Task;
+
+import java.util.List;
+
+public abstract class GetCourseEnrollmentsTask extends
```

Comment:
You should be super confused by this :) I thought I needed to deprecate the other api we are using for this endpoint but ended up not needing.

It still needed to be deprecated but out of scope.